src/Sylius/Bundle/ResourceBundle/ResourceControllerEvents.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Sylius\Bundle\ResourceBundle;
+
+/**
+ * Events that are issued by the resource controller.
+ */
+class ResourceControllerEvents
```

Comment:
Making it static would be a good idea, but unfortunately [it is not possible](https://wiki.php.net/rfc/abstract_final_class). But still, we are able to make it final, and create a private constructor. Hacky way, but there is no better solution.